package.json

```diff
@@ -16,14 +17,13 @@
     "react-copy-to-clipboard": "5.0.1",
     "react-dom": "16.8.3",
     "react-ga": "2.5.6",
-    "react-redux": "5.1.1",
+    "react-redux": "6.0.0",
```

Comment:
there is 6.0.1 from `react-redux`, please update to that please. 
Also please make sure you use yarn v1.13.0 to regenerate `yarn.lock` to avoid `yarn.lock` merge conflicts (Might be over the top but I usually do a `rm -r node_modules/*` before `yarn install`)

Reply:
👍
Yepp! I used v1.13, just messed up with the huge amount of conflicts.
(I guess it was an old branch, a lot behind staging and this caused the merge conflicts -- not not-deleting node_modules. But you're right, I had to delete node_modules to resolve them. Sorry for the incorrect commits. ) 
I'll update redux soon!